### PR TITLE
frontend: Update transfer success tweet text for Polygon release

### PR DESCRIPTION
**Pull request by manuelwedler**
See #1684

Should only be merged shortly before we go live on Polygon.

### CODE REVIEW

**Comment by manuelwedler** [comment]
Yes, already implemented it on top of your PR. Will updated this PR once the other is merged

**Comment by manuelwedler** [comment]
Now this is ready for review.

hint: I removed the `ExtendedTransferData` because it was applied to `TransferData` objects and potentially made a `feeSubAddress` available which does not exist. I think using type guards is the safer approach.

File: frontend/src/components/ShareTweet.vue [hunk 1]
```diff
@@ -25,16 +25,36 @@ const isShareable = computed(() => {
...
+
+Unlock lightning-fast and secure bridging with Beamer today 💪💫  
+https://app.beamerbridge.com/
+`;
+
+  let text: string;
+  if (isSubsidizedTransfer(transfer.value) && transfer.value.fees.uint256.isZero()) {
+    text = subsidizedTransferText;
+  } else {
+    text = defaultText;
```
Comment (GabrielBuragev) [inline frontend/src/components/ShareTweet.vue:54]: Nitpick: We can simply write `return defaultText;` and `return subsidizedTransferText;` here.

**Review by GabrielBuragev — APPROVED** [review]
Thanks for implementing the type guards. Indeed, this is the proper way to do the checks against the different types of transfers.

One small nitpick only.

**Review by GabrielBuragev — CHANGES_REQUESTED** [review]
I just noticed this while testing: https://github.com/beamer-bridge/beamer/issues/1684#issuecomment-1633908079

**Review by GabrielBuragev — APPROVED** [review]
Looks perfect now.

**Pull request merged**